sklearn/tree/tree.py

```diff
@@ -150,6 +153,7 @@ def fit(self, X, y, sample_weight=None, check_input=True,
         """
 
         random_state = check_random_state(self.random_state)
+        beta = self.beta
```

Comment:
I don't think there is a reason to unpack beta here, but that's a minor style thing